luna/featurevis/featurevis.py

```diff
@@ -118,13 +28,34 @@ def visualize_filter(image, model, layer, filter_index, iterations,
     Returns:
         tuple: loss and result image for the process
     """
+    image = tf.Variable(image)
     feature_extractor = get_feature_extractor(model, layer)
+
+    # Temporary method for random choice of
+    # transformation combination
+    choice_num = [0, 1, 2, 3]
+    augmentation = ['noise', 'blur', 'scale']
```

Comment:
Don't we want this to be specified by the user? Can you give an intuition for why this is random?